testing/benchmark/bin/parse_and_send.dart

```diff
@@ -41,15 +41,18 @@ Future<FlutterDestination> connectFlutterDestination() async {
   const String kTokenPath = 'TOKEN_PATH';
   const String kGcpProject = 'GCP_PROJECT';
   final Map<String, String> env = Platform.environment;
+  final bool isTesting = env['IS_TESTING'] == 'true';
```

Comment:
where do we inject the env `IS_TESTING`?

Reply:
In https://github.com/flutter/engine/pull/24022/files#diff-f2e2f624ef84bb33cc75f9c5a09e1c0f9371eb06d5b73ce94fd65ca6ee052b92R27